openfisca_us/variables/gov/irs/credits/cdcc/cdcc_eligible.py

```diff
@@ -0,0 +1,16 @@
+from openfisca_us.model_api import *
+
+
+class cdcc_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "CDCC-eligible"
+    definition_period = YEAR
+    reference = "https://www.law.cornell.edu/uscode/text/26/21#b_1"
+
+    def formula(person, period, parameters):
+        age = person("age", period)
+        max_age = parameters(period).irs.credits.cdcc.eligibility.child_age
+        non_head = ~person("is_tax_unit_head", period)
```

Comment:
For clarity since it's surprising that it includes spouses:
```suggestion
        # Subsection A.
        qualifies_by_age = age < max_age
        # Subsection B (dependent) and C (spouse).
        non_head = ~person("is_tax_unit_head", period)
        disabled = person("incapable_of_self_care", period)
        qualifies_by_disability = non_head & disabled
        return qualifies_by_age | qualifies_by_disability
```